inst/www/shared/shiny_scss/shiny.scss

```diff
@@ -440,3 +440,45 @@ pre.shiny-text-output {
   /* override anything bootstrap sets for `.nav` */
   display: none !important;
 }
+
```

Comment:
If we put this CSS in shiny.css, then this stuff won't work when statically rendered. Maybe this should come in as an HTML dependency on `navSpacer()`?